src/components/Images.vue

```diff
@@ -21,34 +21,82 @@
  -->
 
 <template>
-	<ImageEditor v-if="editing"
-		:mime="mime"
-		:src="src"
-		:fileid="fileid"
-		@close="onClose" />
-
-	<img v-else-if="data !== null"
-		:alt="alt"
-		:class="{
-			dragging,
-			loaded,
-			zoomed: zoomRatio !== 1
-		}"
-		:src="data"
-		:style="imgStyle"
-		@error.capture.prevent.stop.once="onFail"
-		@load="updateImgSize"
-		@wheel="updateZoom"
-		@dblclick.prevent="onDblclick"
-		@mousedown.prevent="dragStart">
+	<div class="image_container">
+		<ImageEditor v-if="editing"
+			:mime="mime"
+			:src="src"
+			:fileid="fileid"
+			@close="onClose" />
+
+		<template v-else-if="data !== null">
+			<img v-if="!livePhotoCanBePlayed"
+				ref="image"
+				:alt="alt"
+				:class="{
+					dragging,
+					loaded,
+					zoomed: zoomRatio !== 1
+				}"
+				:src="data"
+				:style="imgStyle"
+				@error.capture.prevent.stop.once="onFail"
+				@load="updateImgSize"
+				@wheel="updateZoom"
+				@dblclick.prevent="onDblclick"
+				@mousedown.prevent="dragStart">
+
+			<template v-if="livePhoto">
+				<video v-show="livePhotoCanBePlayed"
+					ref="video"
+					:class="{
+						dragging,
+						loaded,
+						zoomed: zoomRatio !== 1
+					}"
+					:style="imgStyle"
+					:playsinline="true"
+					:poster="data"
+					:src="livePhotoSrc"
+					preload="metadata"
+					@canplaythrough="doneLoadingLivePhoto"
+					@loadedmetadata="updateImgSize"
+					@wheel="updateZoom"
+					@error.capture.prevent.stop.once="onFail"
+					@dblclick.prevent="onDblclick"
+					@mousedown.prevent="dragStart"
+					@ended="stopLivePhoto" />
+				<button v-if="width !== 0"
+					class="live-photo_play_button"
+					:style="{left: `calc(50% - ${width/2}px)`}"
+					:disabled="!livePhotoCanBePlayed"
+					:aria-description="t('viewer', 'Play the live photo')"
+					@click="playLivePhoto"
+					@pointerenter="playLivePhoto"
+					@focus="playLivePhoto"
+					@pointerleave="stopLivePhoto"
+					@blur="stopLivePhoto">
+					<PlayCircleOutline v-if="livePhotoCanBePlayed" />
+					<NcLoadingIcon v-else />
+					{{ t('viewer', 'LIVE') }}
```

Comment:
Can we add a translator desc to help them here? :)

Reply:
Possible in vue files ?

Reply:
I have no idea :see_no_evil: